scripts/script.js

```
@@ -0,0 +1,27 @@
+'use strict';
+
+function daysInMonth(month, year) {
+  try {
+    if(month > 11
+    || month < 0
+    || !Number.isInteger(month)
+    || !Number.isInteger(year)
+    || typeof year !== `number`
+    || typeof month !== `number`
+    || month === 0){
+    throw new Error('exception');
+  }
+    const days = new Date(`${year}`, `${month}`, `0`).getDate();
+    return days;
+  }
+  catch(error) {
+    console.log(error);
```

you need to `throw an exception` instead of `return undefined`